packages/slonik/src/dbFilters.ts

```diff
@@ -73,16 +82,17 @@ const applyFiltersToQuery = (
 
   const applyFilters = (
     filters: FilterInput,
-    tableName: string,
+    tableIdentifier: IdentifierSqlToken,
     not = false
   ) => {
     if (filters.AND) {
-      for (const filterData of filters.AND) applyFilters(filterData, tableName);
```

Comment:
`filterData` is a bad variable name for at least 2 reasons:
* 2 words without good reason
* generic word `data`
`conditions` would have been a better choice, but it's a bit long.
`data` is generic, but at least it's short and only 1 word. In this context, where there is little amboguity about what the var represents, this would still be a better choice than `filterData`.